orga/app/components/pagination-control.js

```diff
@@ -0,0 +1,24 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import { inject as service } from '@ember/service';
+
+export default Component.extend({
+  router: service(),
+
+  pagination: null,
+  paginationLink: null,
+
+  nextPage: computed('pagination', function () {
+    return Math.min(this.pagination.page + 1, this.pagination.pageCount);
+  }),
+
+  previousPage: computed('pagination', function () {
+    return Math.max(this.pagination.page - 1, 1);
+  }),
+
+  actions: {
+    changePageSize(pageSize) {
+      this.get('router').replaceWith(this.get('paginationLink'), { queryParams: { pageSize, pageNumber: 1 } });
```

Comment:
Il me semble qu'on peut faire this.router et this.paginationLink directement.

Reply:
en effet